requests.py

```
@@ -0,0 +1,69 @@
+from models import User
+from main import *
+import datetime
+from flask import Flask, jsonify, request
+import logging
+logging.basicConfig(filename='requests.log', filemode='w',
+                    level=logging.DEBUG)
+app = Flask(__name__)
+
+
+@app.route("/api/heart_rate", methods=["POST"])
+def heart_rate_post():
+    """
+    Posts new user with given heart rate
+    """
+    r = request.get_json()
+    try:
+        email = r["user_email"]
+        age = r["user_age"]
+        hr = r["heart_rate"]
+    except KeyError as e:
+        logging.warning("Incorrect JSON input: {}".format(e))
+        return
```

probably want to return a 4xx error code and some sort of error response body